[2025-10-20T16:43:06.970+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-20T16:43:07.007+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Airflow_Lab1_Rithika.load_data_task manual__2025-10-20T16:43:01.188641+00:00 [queued]>
[2025-10-20T16:43:07.016+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Airflow_Lab1_Rithika.load_data_task manual__2025-10-20T16:43:01.188641+00:00 [queued]>
[2025-10-20T16:43:07.018+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-20T16:43:07.032+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_data_task> on 2025-10-20 16:43:01.188641+00:00
[2025-10-20T16:43:07.047+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=180) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-20T16:43:07.053+0000] {standard_task_runner.py:63} INFO - Started process 199 to run task
[2025-10-20T16:43:07.051+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Airflow_Lab1_Rithika', 'load_data_task', 'manual__2025-10-20T16:43:01.188641+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/***_lab1_dag.py', '--cfg-path', '/tmp/tmpgo4m5ild']
[2025-10-20T16:43:07.057+0000] {standard_task_runner.py:91} INFO - Job 17: Subtask load_data_task
[2025-10-20T16:43:07.156+0000] {task_command.py:426} INFO - Running <TaskInstance: Airflow_Lab1_Rithika.load_data_task manual__2025-10-20T16:43:01.188641+00:00 [running]> on host bddbc10ace4f
[2025-10-20T16:43:07.249+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='rithika' AIRFLOW_CTX_DAG_ID='Airflow_Lab1_Rithika' AIRFLOW_CTX_TASK_ID='load_data_task' AIRFLOW_CTX_EXECUTION_DATE='2025-10-20T16:43:01.188641+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-20T16:43:01.188641+00:00'
[2025-10-20T16:43:07.251+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-20T16:43:07.290+0000] {logging_mixin.py:188} INFO - âœ… Data loaded successfully.
[2025-10-20T16:43:07.297+0000] {python.py:237} INFO - Done. Returned value was: {'feature1': {0: 5, 1: 15, 2: 25, 3: 35, 4: 45, 5: 55}, 'feature2': {0: 2, 1: 8, 2: 18, 3: 32, 4: 38, 5: 50}}
[2025-10-20T16:43:07.298+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-20T16:43:07.321+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Airflow_Lab1_Rithika, task_id=load_data_task, run_id=manual__2025-10-20T16:43:01.188641+00:00, execution_date=20251020T164301, start_date=20251020T164307, end_date=20251020T164307
[2025-10-20T16:43:07.351+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-20T16:43:07.375+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-20T16:43:07.377+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
